fuzz/README.md

```diff
@@ -0,0 +1,18 @@
+This directory contains fuzzing targets to verify implementation correctness: 
+
+- `roundtrip_*` targets check that we can successfully decode what we've encoded.
+- `decompress_*` targets check that we don't panic or abort on decoding a crafted file.
+- `compare_*` targets check that we produce identical output to liblzma on decompression.
+
+The command to run fuzzer is:
+
+`cargo +nightly fuzz run --release -s none <fuzzing_target>`
+
+For example,
+
+`cargo +nightly fuzz run --release -s none compare_xz`
+
+We use `-s none` because this crate does not contain unsafe code, so we don't
+need sanitizers to detect memory or concurrency errors for us.
```

Comment:
In the compare case, shouldn't we sanitize the liblzma part? Or would the sanitizing be scoped to the Rust binding wrapper only, which is likely not so interesting, but not applied to the liblzma itself (being compiled independently by a C/C++ compiler)?

Reply:
> Or would the sanitizing be scoped to the Rust binding wrapper only, which is likely not so interesting, but not applied to the liblzma itself 

Yes, that's the case.

Besides, I'm not particularly interested in fuzzing liblzma itself because [Google already fuzzes it](https://bugs.chromium.org/p/oss-fuzz/issues/list?q=-status%3AWontFix%2CDuplicate%20-component%3AInfra%20proj%3Dlzma&can=1).